### ISSUE TITLE
Update Pinta PPA with recent versions

### ISSUE THREAD

**Issue by MikeNavy** [PintaProject/Pinta#738]
**Description**
Make available Pinta 2.1.1 and further versions as debs, in Pinta PPA https://launchpad.net/~pinta-maintainers/+archive/ubuntu/pinta-stable, for Jammy and further Ubuntu versions.

**Additional context**
From Ubuntu Jammy, 22.04 LTS, and further versions (Mantic; Noble future 24.04 LTS...), "dotnet-sdk-8.0" is available as a package from Ubuntu repository. See https://pkgs.org/download/dotnet-sdk-8.0

This makes possible to have Pinta 2.1.1 available from a launchpad PPA, since all the dependencies can be found in Ubuntu repository and since the installation of .Net 8.0 from Microsoft is no longer required.

**Comment by cameronwhite** [comment]
Yep, having dotnet available definitely helps. I think the additional problem is that the PPA builder doesn't have internet access, so Pinta's dependencies on Nuget packages involve some extra complexity to get those bundled up and working

The maintainer of this PPA (https://xtradeb.net/apps/pinta/) worked out how to solve those issues, so similar changes could be ported over to the official PPA in the future. I'd suggest using that PPA for now though!

**Comment by Fr-Dae** [comment]
any news for 24.04 ?
23.04 ppa don't work

**Comment by cameronwhite** [comment]
There are no immediate plans since I don't have the development time to work on maintaining more packages. I'd suggest using the PPA linked in the previous comment, or using the snap / flatpak packages

**Comment by Fr-Dae** [comment]
I can't understand why people do this to themselves, it's so bad with snap, and it adds so much unnecessary and heavy dependency :/.

**Comment by cameronwhite** [comment]
I would suggest contacting the maintainers of https://xtradeb.net/apps/pinta/ to ask them to update their PPA for 24.04 (there are already packages for 23.10 and 22.04 by the look of it)

**Comment by Fr-Dae** [comment]
22.04 use python 3.10 
23.04 use python 3.11
so i can't use old ppa :/

**Comment by cameronwhite** [comment]
Their webpage has links to their contact email and bug tracker

**Comment by JGCarroll** [comment]
The Pinta snap is one of the best snaps going!

It's also the smallest version of Pinta, at just 34.4 MB installation space used. The flatpak for example is 100MB. The Windows installer is 50MB download and 220MB once installed. (N.B, this assumes we're removing duplicated packages, which both the Flatpak and Snap do).

From Ubuntu 22.04 where you'd already have the Gnome snap and the rest installed, it literally really is 34.MB installed.

If it isn't a satisying experience, I'll personally confront the guy who made it ;).

**Comment by MestreLion** [comment]
Thumbs up on this initiative!

Coming from Ubuntu 18.04 to 24.04, I was surprised that Pinta was not on the repositories, and not even released as a DEB anymore. If the dotnet issues were sorted out and there's a un-official PPA, it means it _can_ be packaged as a DEB, which is great!

Sure, Snap is a viable alternative (IMHO better than Flatpak, but that's debatable), but no sandbox solution will beat the simplicity and availability of good old DEBs and `apt` repositories.

Hope this goes forward!

**Comment by Fr-Dae** [comment]
Hello, 
Ubuntu 24.04 and 26.04 are both LTS releases, 
so why on earth are only non-LTS versions (21.10, 23.04) 
available on  

- **unstable 2.1.1** for ubuntu 23.04 https://launchpad.net/~pinta-maintainers/+archive/ubuntu/pinta-daily?field.series_filter=lunar
- **stable 1.7.1** for ubuntu 21.10 https://launchpad.net/~pinta-maintainers/+archive/ubuntu/pinta-stable?field.series_filter=impish

I would also like to point out that this documentation is out of date 
https://doc.ubuntu-fr.org/pinta

il semble que la seule source offrant un .deb est ici 

```properties
wget https://launchpad.net/~xtradeb/+archive/ubuntu/apps/+files/xtradeb-apt-source_0.4_all.deb && sudo apt install ./xtradeb-apt-source_0.4_all.deb
```

@xtradeb
@jhonny-oliveira
@iskunk

If you are reading this message, your assistance would be much appreciated

**Comment by cameronwhite** [comment]
Those `pinta-maintainers` PPAs were unmaintained since they cannot build Pinta 2.x or later. I've disabled them to avoid future confusion

**Comment by Fr-Dae** [comment]
ok, so what are the solution here ?

**Comment by cameronwhite** [comment]
> ok, so what are the solution here ?

If you want an apt repository, the recommendation is to use https://xtradeb.net/apps/pinta/
